compiler/dyno/doc/CMakeLists.txt

```diff
@@ -22,13 +22,19 @@ if(DOXYGEN_FOUND)
   # doxygen config-file settings can be set here by prefixing with DOXYGEN_
   set(DOXYGEN_GENERATE_XML YES)
   set(DOXYGEN_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/doxygen")
+  set(DOXYGEN_INCLUDE_PATH ${CHPL_MAIN_INCLUDE_DIR})
+
   # the following exclude avoids errors in the sphinx build
-  set(DOXYGEN_EXCLUDE "${CHPL_MAIN_INCLUDE_DIR}/chpl/AST/uast-classes-list.h")
+  #set(DOXYGEN_EXCLUDE "${CHPL_MAIN_INCLUDE_DIR}/chpl/AST/uast-classes-list.h";
+  #   )
```

Comment:
these commented-out things are no longer needed but I should remove the commented-out code before merging